.travis.yml

```diff
@@ -154,15 +160,6 @@ matrix:
           compiler: clang
         - os: osx
           compiler: gcc
-        - arch: arm64
-          os: osx
-        # arm64 times out when using -fsanitize=address
-        # The amd64 build should be sufficient to test this.
-        # Note: the env line must exactly match the env line from the build
-        # matrix above which contains the `-fsanitize=address` option.
-        # TODO: come up with a better way of doing this that is less error prone.
-        - arch: arm64
-          env: CONFIG_OPTS="no-asm -Werror --debug no-afalgeng no-shared enable-rc5 enable-md2 -fsanitize=address" LSAN_OPTIONS="report_objects=1"
```

Comment:
This means we're losing the asan ARM build doesn't it?
Is there a possibility of keeping it by fiddling without options?  e.g. ditch the `no-asm` or `--debug` perhaps?

Reply:
This build was already excluded. Is there any reason why this particular build would be so much useful when we have asan build for x86_64?

Reply:
With `no-asm` I guess they'll be pretty much the same.....

I err towards more testing rather than less.